src/main/java/Group15/ExerciseDetailsView.java

```diff
@@ -0,0 +1,106 @@
+package Group15;
+
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.*;
+import javafx.geometry.Pos;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontWeight;
+
+public class ExerciseDetailsView {
+
+    private static String exerciseTitle = "Bench Press";
```

Comment:
Perhaps we should have a exercise class that holds these variables

Reply:
You would almost think someone has already made that :O

Reply:
Then we should use that:)